Source/Core/DolphinQt/Debugger/RegisterWidget.cpp

```diff
@@ -71,6 +71,7 @@ void RegisterWidget::CreateWidgets()
 
   m_table->setColumnCount(9);
 
+  m_table->horizontalHeader()->setVisible(false);
```

Comment:
I can't test right now unfortunately, but if I recall, doesn't this prevent resizing the columns?  Because if it does, I am not sure this woudl be a good idea.

Just to make sure.

Reply:
Oh damn, you're right. Sorry wasn't thinking about that at all :disappointed: